pyshelf/search/sorter.py

```diff
@@ -0,0 +1,39 @@
+from distutils.version import LooseVersion
+from pyshelf.search.sort_type import SortType
+from pyshelf.search.sort_flag import SortFlag
+
+
+class Sorter(object):
+    def sort(self, data, sort_criteria):
+        """
+            Sorts results based on sort criteria.
+
+            Args:
+                data(List[dict]): Data to sort using given sort criteria.
```

Comment:
Apparently not. I can create a task for this. Figured it wasn't worth putting in this hot fix.